ts/score/peer-score.ts

```diff
@@ -30,6 +30,18 @@ export class PeerScore {
    * IP colocation tracking; maps IP => set of peers.
    */
   peerIPs: Map<string, Set<string>>
+  /**
+   * Cache score up to decayInterval if topic stats are unchanged.
+   */
+  scoreCache: Map<string, number>
+  /**
+   * Flag to mark a peer score cache valid or not.
+   */
+  scoreCacheValid: Map<string, boolean>
```

Comment:
yeah that's a good idea 👍 , we can save 1 variable. Instead of
```
scoreCacheValid.set(id, false)
```
do
```
scoreCacheTime.delete(id)
```

when calling `score(id)`, check for the existence of `scoreCacheTime` first

Reply:
Update: I changed from `scoreCacheTime` to `scoreCacheUntil` as the other comment